main/nfs-utils/nfs-utils.post-upgrade

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+ver_new="$1"
+ver_old="$2"
+
+if [ "$(apk version -t "$ver_old" '2.3.2-r2')" = '<' ]; then
+	# migrate owner from root: -> statd:
+	chown statd: /var/lib/nfs \
```

Comment:
@ncopa how to handle that rpc.statd should not be running when doing this? are we "allowed" to restart processes during upgrade?

Reply:
Generally speaking I would *personally* prefer avoiding automigration magic in post install scripts and instead print a message informing the user of the necessary migration steps.

Reply:
Given that the old process is running as root, does this chown affect it at all?